shared/login/signup/request-invite/index.js

```diff
@@ -0,0 +1,55 @@
+// @flow
+import * as React from 'react'
+import * as Constants from '../../../constants/signup'
+import {Text, Icon, Input, WaitingButton} from '../../../common-adapters'
+import Wrapper from '../wrapper'
+
+type Props = {|
+  emailError: string,
+  onBack: () => void,
+  onSubmit: (name: string, email: string) => void,
+  nameError: string,
+|}
+type State = {|
+  name: string,
+  email: string,
+|}
+
+class RequestInvite extends React.Component<Props, State> {
```

Comment:
Do we still need the invite flow at all?

Reply:
we do. the invite codes are determined by the server so it can always opt to not give you one and you have to go down this flow